src/OMSimulatorLib/System.cpp

```diff
@@ -1168,3 +1155,47 @@ oms_status_enu_t oms3::System::updateDependencyGraphs()
 
   return oms_status_ok;
 }
+
+oms_status_enu_t oms3::System::importFromSSD_ConnectionGeometry(const pugi::xml_node& node, const ComRef& crefA, const ComRef& crefB)
```

Comment:
Should this be better moved to `oms2::ssd:ConnectionGeometry`? And it should probably be copied to `oms3::ssd:ConnectionGeometry`.

Reply:
I added `oms2::ssd::ConnectionGeometry::importFromSSD` but I think we still need this function. Most of the functionality is moved to its respective class.